src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadManifestUpdater.cs

```diff
@@ -21,6 +21,8 @@ internal interface IWorkloadManifestUpdater
         IEnumerable<WorkloadId> GetUpdatableWorkloadsToAdvertise(IEnumerable<WorkloadId> installedWorkloads);
 
         void DeleteUpdatableWorkloadsFile();
+
+        void DownloadWorkloadSet(string version, DirectoryPath? offlineCache);
```

Comment:
The workload manifest updater doesn't need to download a specific version of a workload set.  Its purpose is to check for the latest version available.  The installer implementations download (via NuGetPackageDownloader) the actual package to be installed (which will differ depending on whether the installer is the File-based or MSI-based one).

Reply:
This is for --version, in which case the user does specify an exact version they want. If they just do `dotnet workload update`, it goes through a different path that ultimately picks the highest version.